tests/integration/targets/aci_node_mgmt_epg_to_contract/tasks/main.yml

```diff
@@ -0,0 +1,353 @@
+# Test code for the ACI modules
+# Copyright: (c) 2024, Faiz Mohammad (@faizmoh) <faizmoh@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
can you convert to this format: https://github.com/CiscoDevNet/ansible-aci/blob/master/tests/integration/targets/aci_esg_to_contract/tasks/main.yml